test/cls_test.py

```diff
@@ -705,7 +705,7 @@ def test_web_cls(client):
         assert c.asgi.web_url == "http://asgi.internal"
 
 
-handler_app = App("handler-app")
+handler_app = App("handler-app", include_source=True)
```

Comment:
Do you intend not to remove this later or did you just get tired of writing the comment everywhere? 😁

Reply:
It was one of the instances where my regex didn't trigger so I made the edit manually :D